thoth/storages/graph/postgres.py

```diff
@@ -4392,6 +4392,7 @@ def sync_analysis_result(self, document: dict) -> None:
                         os_name=os_name,
```

Comment:
why are some names with `-` and some with `_` ?

Reply:
I introduced this naming issue back in early package-extract days and it is still present :). We should switch to `_` for consistency (fix package-extract and syncing logic).

Reply:
> I introduced this naming issue back in early package-extract days and it is still present :). We should switch to `_` for consistency (fix package-extract and syncing logic).

Anyway, I think this can be merged and inconsistencies in `-` and `_` can be addressed in a separate PR.